app/models/event.rb

```diff
@@ -21,7 +21,7 @@ def ical
     e
   end
 
-  def as_json
+  def as_json(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `as_json(*)` if you want the method to accept any arguments but don't care about them.